src/index.js

```diff
@@ -61,13 +61,14 @@ function rollem (configs, options) {
       if (err) {
         watcher.emit('failed', err)
       } else {
-        gazeWatcher.on('all', () => {
-          watcher.emit('changed')
-          buildBundles(configs).then(() => watcher.emit('rolled'))
+        gazeWatcher.on('all', (_event, filepath) => {
```

Comment:
The rule, which you mentioned checks arguments from right to left. You cannot give parameters to a function and leave out something from the left, but you can ignore things from the right. The rule only tries to narrow down the second case:

```javascript
const fn = (a, b, c) => {
  console.log('I only care about b and c, please don\'t define a', b, c);
};

fn('you cannot skip defining me', 12, 34)
```

( [Maybe python allows something like this](http://stackoverflow.com/q/21694536/1806628), but JS doesn't )